### PR TITLE
`microbe-contract.md` w/ `README` to preview/generate w/ `quarto` (#5)

### LINKED ISSUE #5: [Ops]: Create Common Contract Templates

### PR DESCRIPTION
**Pull request by phoward38**
*Have you read the [Contributing Guidelines](https://github.com/noclocks/.github/blob/main/CONTRIBUTING.md)?*

Issue #5

## Changes description

- New `microbe-contract.md`
- `README` for preview/generation w/ `quarto`
- `.gitignore` generated files/directories

### CODE REVIEW

File: .gitignore
Comment (jimbrig) [inline .gitignore:1]: just add `docs/documents/contacts/microbe/*` or a separate `.gitignore` in that folder w/ `*` and `!.gitignore`, etc.

**Review by jimbrig — APPROVED** [review]
Cool - can alter the gitignore slightly

**Comment by jimbrig** [comment]
@phoward38 I would also change file extension to use `qmd` instead of `md`

**Comment by phoward38** [comment]
@jimbrig yea, I started working off the existing `md` template then VSCode's built in generator is `quarto`

**Pull request merged**